helm-charts/stable/binfmt/values.yaml

```diff
@@ -0,0 +1,19 @@
+# Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+images:
+  binfmtInstallerImage: public.ecr.aws/eks-distro-build-tooling/binfmt-misc:qemu-v6.1.0
+  binfmtIdleImage: public.ecr.aws/eks-distro-build-tooling/builder-base:latest
```

Comment:
possibly overkill? But better to avoid Dockerhub rate limiting

Reply:
Another option is to use the eks-distro-base image

Reply:
Yeah use eks distro base